openlibrary/plugins/upstream/models.py

```diff
@@ -678,7 +678,7 @@ def get_sorted_editions(self):
             edition_keys = web.ctx.site.things(db_query)
 
         editions = web.ctx.site.get_many(edition_keys)
-        editions.sort(key=lambda ed: ed.get_publish_year(), reverse=True)
+        editions.sort(key=lambda ed: ed.get_publish_year() or 0, reverse=True)
```

Comment:
`-10 > None` is also true, so we want this to default to negative infinity

Reply:
How many works do we have that were published before Gutenberg? ;-)

Reply:
:D Haha probably none, but easier to keep things consistent :P